sklearn/preprocessing/data.py

```diff
@@ -503,11 +511,10 @@ class StandardScaler(BaseEstimator, TransformerMixin):
 
     See also
     --------
-    :func:`sklearn.preprocessing.scale` to perform centering and
-    scaling without using the ``Transformer`` object oriented API
+    scale: Equivalent function without the object oriented API
 
-    :class:`sklearn.decomposition.RandomizedPCA` with `whiten=True`
-    to further remove the linear correlation across features.
+    :class:`sklearn.decomposition.PCA`
+        Further removes the linear correlation across features with 'whiten=True'.
```

Comment:
@hlin117 `RandomizedPCA` is now deprecated (its features have been included into the PCA class). Deprecated class and function should no longer be mentioned in the doc (beside the changelog).